testing/code/test_terminal_writer.py

```diff
@@ -0,0 +1,45 @@
+import re
+from io import StringIO
+
+import pytest
+from _pytest._io import TerminalWriter
+
+
+# TODO: move this and the other two related attributes from test_terminal.py into conftest as a
```

Comment:
Thinking of moving this to a `fixture` and reuse on tests appropriately.

Reply:
Should have a look at the constant declaration in colorama

Reply:
Done all, except that I could not find the exact equivalent of 'hl-reset' code using colorama
constants though.

Reply:
Oh `colorama` is not a dependency on Linux, so I've moved this back to constants; not sure it is worth the extra dependency just because of this.